messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/messaging/kafka/reactive/streams/DevModeRedPandaDevServiceUserExperienceIT.java

```diff
@@ -15,18 +18,18 @@
 @QuarkusScenario
 public class DevModeRedPandaDevServiceUserExperienceIT {
 
-    private static final String RED_PANDA_VERSION = "v22.3.4";
-    private static final String RED_PANDA_IMAGE = "vectorized/redpanda";
+    private static final String RED_PANDA_VERSION = getImageVersion("redpanda.image");;
+    private static final String RED_PANDA_IMAGE = getImageName("redpanda.image");
 
     @DevModeQuarkusApplication
     static RestService app = new RestService()
             .withProperty("quarkus.kafka.devservices.enabled", Boolean.TRUE.toString())
-            .withProperty("quarkus.kafka.devservices.image-name", String.format("%s:%s", RED_PANDA_IMAGE, RED_PANDA_VERSION))
+            .withProperty("quarkus.kafka.devservices.image-name", "${redpanda.image}")
             .onPreStart(s -> DockerUtils.removeImage(RED_PANDA_IMAGE, RED_PANDA_VERSION));
 
     @Test
     public void verifyIfUserIsInformedAboutRedPandaDevServicePulling() {
-        app.logs().assertContains("Pulling docker image: vectorized/redpanda");
+        app.logs().assertContains("Pulling docker image: redpandadata/redpanda");
```

Comment:
I suppose this is fine, but can you provide me with context why this changed? Is it just renamed product as version have changed?

Reply:
vectorized/redpanda repo was deprecated and was migrated to redpandadata/redpanda

Reply:
Thank you